steampy/client.py

```diff
@@ -278,7 +278,7 @@ def make_offer_with_url(self, items_from_me: List[Asset], items_from_them: List[
         }
         headers = {'Referer': self.COMMUNITY_URL + urlparse.urlparse(trade_offer_url).path,
                    'Origin': self.COMMUNITY_URL}
-        response = self._session.post(url, data=params, headers=headers).json()
+        response = self._session.post(url, data=params, headers=headers).json() or {}
```

Comment:
I think the exception is expected behaviour.
You will have to check it anyway, what will simplify getting {} instead of exception?